### ISSUE TITLE
Notification bug

### ISSUE THREAD

**Issue by KParthSingh** [vfsfitvnm/ViMusic#985]
### Steps to reproduce the bug

Just play any random song

### Expected behavior

Look at the screenshot the weight of background is too big

### Actual behavior

.

### Screenshots/Screen recordings

![Screenshot_2023-03-03-18-04-47-697_lockscreen](https://user-images.githubusercontent.com/99123839/222922008-df6a0aee-da07-4ee7-9b86-9e59bc60e228.jpg)

### ViMusic version

0.5.5

### Android version

Android 10

### Additional information

Nothing. It is what it is...

**Comment by KParthSingh** [comment]
Please note: Vimusic version is 0.5.4*

**Comment by 100lankiParth** [comment]
I didn't know how to give a suggestion, I barely use github and didn't want to raise issue for just info. 

Is there any way to make this app a frontend for spotify instead of youtube music? 

I like the ui, the minimalistic size and functionality of playing the songs offline from cached. But its just that there are many podcasts and songs that are on spotify. Yes some could argue that the songs can also be found on youtube but I just wanted a similar small app for spotify.

I don't want to be presumed as demanding in any way, I just like the app greatly and was hoping you could tell me if it is possible or am I hoping for something very hard to do at the backend

**Comment by Swadhin05** [comment]
Yes I also face the same issue. Notification card size is inconsistent

**Comment by vfsfitvnm** [comment]
What ROM (es OneUI, MIUI etc) are you using?
Also, could you provide a screenshot of how it should look like?

**Comment by KParthSingh** [comment]
> 

Here's my phones about
![Screenshot_2023-03-05-19-05-30-949_com android settings](https://user-images.githubusercontent.com/99123839/222963801-8a811ab1-058b-4966-a3a3-f69b46711a4a.jpg)

By turn on show song cover nothing happens, it remains as it is. Btw In the first screenshot (on starting of this thread) this option was already enabled.

**Comment by KParthSingh** [comment]
> I didn't know how to give a suggestion, I barely use github and didn't want to raise issue for just info.
> 
> Is there any way to make this app a frontend for spotify instead of youtube music?
> 
> I like the ui, the minimalistic size and functionality of playing the songs offline from cached. But its just that there are many podcasts and songs that are on spotify. Yes some could argue that the songs can also be found on youtube.
> 
> I don't want to be sound demanding in any way, I just like the app greatly and was hoping you could tell me if it is possible or am I hoping for something very hard to do at the backend

No i there isn't a Spotify client. It's just for yt

**Comment by vfsfitvnm** [comment]
> > Update: i got to know that turning off Show song cover setting fix this issue.
> 
> Hey where is this setting ? I am facing the similar problem, the song notification image is stretched resolution
> 
> the "show song cover" isn't in Appearance setting ?

That setting is hidden since Android 13, because the new notification system always requires that setting to be on. However, it looks like MIUI likes to mess things up, and it still uses the old behavior... I will just show that setting despite the Android version then...

**Comment by KParthSingh** [comment]
Update 2: last time i told you that issue is due to setting but now it seems that its not related. For some reason even tho setting is off its still happening.

**Comment by vfsfitvnm** [comment]
Thanks for the update. Is this problem ViMusic specific?

**Comment by KParthSingh** [comment]
> Thanks for the update. Is this problem ViMusic specific?

Yeah

**Comment by arijitroy454** [comment]
> Update 2: last time i told you that issue is due to setting but now it seems that its not related. For some reason even tho setting is off its still happening.


Does not fix even with that setting turned off.
I'm also Miui user. Miui 12.5.1 Android 10.
And that setting does nothing for me (does not do what it claims to)

It happens for me, and is reproducible, it happens after a reboot, when vimusic is started for the first time and a music is played from some playlist.
